suite-native/biometrics/src/isBiometricsFeatureAvailable.ts

```diff
@@ -3,5 +3,5 @@ import { getEnrolledLevelAsync, SecurityLevel } from 'expo-local-authentication'
 export const getIsBiometricsFeatureAvailable = async () => {
     const enrolledLevelAsync = await getEnrolledLevelAsync();
 
-    return enrolledLevelAsync !== SecurityLevel.NONE;
+    return enrolledLevelAsync === SecurityLevel.BIOMETRIC;
```

Comment:
Why is this change necessary? I thought the issue was caused by `useEffect` in the hook?

Reply:
I guess it won't work for touch id with this change, have you tested that?

Reply:
I've simplified how things work once more and reverted this part.
17d04cf14b